elastalert/ruletypes.py

```diff
@@ -301,6 +301,7 @@ def check_for_match(self, key, end=False):
             event = self.occurrences[key].data[-1][0]
             if self.attach_related:
                 event['related_events'] = [data[0] for data in self.occurrences[key].data[:-1]]
+            event['count'] = self.occurrences[key].data[-1][1]
```

Comment:
why is this required in frequency rule type?

Reply:
That was from different PR, by mistake got added here. Removed it.